base/commands/viridian/custom_class_delete.go

```diff
@@ -0,0 +1,59 @@
+package viridian
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+)
+
+type CustomClassDeleteCmd struct{}
+
+func (cmd CustomClassDeleteCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("delete-custom-class [cluster-ID/cluster-name] [file-name/artifact-id] [flags]")
```

Comment:
Let's use the same convention for `ID`. Since we have `cluster-ID`, could you use `artifact-ID` here and other places?